manifests/init.pp

```diff
@@ -10,14 +10,16 @@
   Boolean $debug = false,
   Boolean $trace = false,
   Boolean $announce_cluster = false,
-  String $binpath = "/usr/sbin/gnatsd",
-  String $configdir = "/etc/gnatsd",
-  String $piddir = "/var/run",
+  Stdlib::Absolutepath $binpath = "/usr/sbin/gnatsd",
```

Comment:
we will need to add the stdlib dependency to metadata.json, soonish Puppet wont find these things without the dependency